.github/CONTRIBUTING.md

```diff
@@ -34,14 +37,24 @@ Please fill in the description following the template.
 
 9.  In order to be able to publish packages on `npm`, make sure you are logged in. `npm whoami` will show your npm username if you are logged in. Otherwise, please run `npm login`.
 
-10. To publish the latest changes run `npm run publish` **Do not run `npm publish`!**
+10. To publish the latest changes run `npm run publish` :warning: **Do not run `npm publish`!**
 
 11. The CLI will ask for a confirmation about the new package versions. Please verify them carefully before accepting.
 
 12. Once you have accepted, the script will start publishing the packages. Go to your `npm` user profile and organisation `@ecl` and double-check packages have actually been published.
 
 13. Push the locally-synched `v2` to the remote. This marks the release as complete.
 
-14. Finally, create a GitHub Release with the same text as the changelog generated at step 3. Add "useful links" section. Congratulations!
+14. Finally, create a GitHub Release from branch `v2` or `v2-dev` (they are equal at this point), with the following text:
```

Comment:
It's true that the branches are equal at this point, however it'd be probably better to mention v2, as it should be the release branch.